src/Profiler.php

```diff
@@ -155,7 +155,7 @@ public function setEventManager(EventManagerInterface $eventManager)
      *
      * @return EventManagerInterface
      */
-    public function getEventManager()
+    public function getEventManager(): EventManagerInterface
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!